library/ssl_tls13_client.c

```diff
@@ -148,33 +148,67 @@ static int ssl_tls13_generate_and_write_ecdh_key_exchange(
                 unsigned char *end,
                 size_t *out_len )
 {
-    int ret = MBEDTLS_ERR_ERROR_CORRUPTION_DETECTED;
-    const mbedtls_ecp_curve_info *curve_info =
-        mbedtls_ecp_curve_info_from_tls_id( named_group );
+        psa_status_t status = PSA_ERROR_GENERIC_ERROR;
+        int ret = MBEDTLS_ERR_SSL_FEATURE_UNAVAILABLE;
+        psa_key_attributes_t key_attributes;
+        unsigned char own_pubkey[MBEDTLS_PSA_MAX_EC_PUBKEY_LENGTH];
+        size_t own_pubkey_len;
+        mbedtls_ssl_handshake_params *handshake = ssl->handshake;
+        size_t ecdh_bits = 0;
+
+        MBEDTLS_SSL_DEBUG_MSG( 1, ( "Perform PSA-based ECDH computation." ) );
+
+        // --- Just for now --- !!!
```

Comment:
This needs to be resolved before the PR can be merged. Applications should be calling this so we don't need to call it in the library. I guess we'll want to adjust the guards around the calls in `programs/ssl/*.c` (similarly to what you did in the previous commit).

Reply:
Yes, I'm looking into this right now.